src/ontology/mondo-edit.obo

```diff
@@ -484878,21 +484878,17 @@ property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/1607/crawfur
 
 [Term]
 id: MONDO:0022900
-name: athyreotic congenital hypothyroidism
-def: "Congenital hypothyroidism in which fetal deficiency was severe because of complete absence (athyreosis) of the gland. Physical features may include a larger anterior fontanel, persistence of a posterior fontanel, an umbilical hernia, and a large tongue (macroglossia)" [Wikipedia:Congenital_hypothyroidism]
+name: obsolete athyreotic congenital hypothyroidism
+def: "OBSOLETE. Congenital hypothyroidism in which fetal deficiency was severe because of complete absence (athyreosis) of the gland. Physical features may include a larger anterior fontanel, persistence of a posterior fontanel, an umbilical hernia, and a large tongue (macroglossia)" [Wikipedia:Congenital_hypothyroidism]
 comment: Reason of obsoletion: out of scope - MONDO:excludeHistoricalDisease. Term to consider: athyreosis-MONDO:0019855
-subset: gard_rare {source="GARD:1610", source="MONDO:GARD"}
-subset: obsoletion_candidate
-subset: rare
 synonym: "cretinism athyreotic" RELATED DEPRECATED [GARD:0001610]
-xref: GARD:1610 {source="MONDO:GARD"}
-is_a: MONDO:0002254 {source="MONDO:Redundant", source="https://orcid.org/0000-0002-6601-2165"} ! syndromic disease
-is_a: MONDO:0018612 {source="MONDO:Redundant"} ! congenital hypothyroidism
-relationship: has_characteristic MONDO:0021140 {source="https://orcid.org/0000-0002-4142-7153"} ! congenital
+xref: GARD:1610 {source="MONDO:obsoleteEquivalent"}
+property_value: IAO:0000231 OMO:0001000 {source="MONDO:excludeHistoricalDisease"}
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/4069" xsd:anyURI
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7357" xsd:anyURI
 property_value: IAO:0006012 "2024-06-01" xsd:string
```

Comment:
schedule for obsoletion date should be removed

Reply:
done